src/Map/Navigation/Path.java

```diff
@@ -268,7 +278,7 @@ private void createPathRandom(MapNode start, MapNode end, boolean vFlag) throws
      * @param vFlag
      * @throws PathFindingException
      */
-    private void createPathBreadthFirst(MapNode start, MapNode end, boolean vFlag) throws PathFindingException {
+    private void createPathBreadthFirst(MapNode start, MapNode end, boolean vFlag, boolean useStairs) throws PathFindingException {
```

Comment:
Really not a big fan of the vflag. Comment out the functions that use it  (but leave them there) so that the function can be called with just the parameters (start, end, useStairs)